docs/examples/use_cases/pytorch/resnet50/main.py

```diff
@@ -20,7 +20,7 @@
 
 try:
     from nvidia.dali.plugin.pytorch import DALIClassificationIterator, LastBatchPolicy
-    from nvidia.dali.pipeline import Pipeline
+    from nvidia.dali.pipeline import pipeline
```

Comment:
have we decided that we want to replace examples?

Reply:
I'm replacing subset of examples